### ISSUE TITLE
Development status

### ISSUE THREAD

**Issue by ajnelson-nist** [casework/CASE-Implementation-UFED-XML#1]
After discussion on this morning's call, the criteria for a `casework` Github project self-designating as "Beta" development status changed to include running tests with Continuous Integration.

This project currently designates as "Beta" status, but does not meet the (new) requirement of running tests with Continuous Integration.  In some sense it does not meet the (old) requirement of running tests either; the project's output is *demonstrated* with the `/json` directory of sample data, but not tested with either ground-truth content review (e.g. a test that there should be exactly *n* `observable:Message` objects), or by running through SHACL validation.

Can this repository please either:
* Downgrade its development status to "Alpha"; or,
* Add some scripted testing of the code base.  I'm not sure how this project would be *run* through CI without sample data being incorporated or downloaded, but I think it would be helpful to see the `/json` directory run through `case_validate`.  @kchason , would you be able to add the validation action here?

Adding CI to this project will make for a more dynamic review process, so I encourage testing be incorporated.

**Comment by kchason** [comment]
Yes, I can add the validation action for that, and setup a basic CI pipeline that can be extended as appropriate as well

Regarding the CI testing, it would be ideal to have a publicly sharable Cellebrite Physical Analyzer (I'm assuming that's the target of this project now) XML report that we could store here and have it run through the parser and test the output, but that's a little more involved.

**Comment by ajnelson-nist** [comment]
I think there is a shareable file somewhere between the Owl Trafficking scenario, the Crossover scenario, and the Hickman Android images on Digital Corpora.  After October 1 I can index that file on CASE-Corpora and set up a download for it.
